excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CaseFieldParserTest.java

```diff
@@ -41,15 +42,12 @@ public void setup() {
         definitionSheets.put(SheetName.CASE_FIELD.getName(), definitionSheet);
     }
 
-    @Test(expected = SpreadsheetParsingException.class)
+    @Test
     public void shouldFail_whenNoFieldsAreDefinedForCaseType() {
-        try {
-            caseFieldParser.parseAll(definitionSheets, caseType);
-        } catch (SpreadsheetParsingException ex) {
-            Assertions.assertThat(ex).hasMessageContaining(
+        SpreadsheetParsingException ex = assertThrows(SpreadsheetParsingException.class, () ->
+            caseFieldParser.parseAll(definitionSheets, caseType));
+        Assertions.assertThat(ex).hasMessageContaining(
```

Comment:
Two different versions of `assertThat()` are used in this class (`org.hamcrest.MatcherAssert.assertThat` and `org.assertj.core.api.Assertions.assertThat`).  Can this be simplified to use just one?  The assertj one is the least frequently used one.

Reply:
Class simplified to use one version of assertThat().